tests/negative/security/tls.go

```diff
@@ -185,6 +247,46 @@ func AppendConfigCustomCertHTTP() types.Test {
 	}
 }
 
+func AppendConfigCABundleCertHTTP() types.Test {
+	name := "tls.config.merge.http.cabundle"
+	in := types.GetBaseDisk()
+	out := types.GetBaseDisk()
+	config := fmt.Sprintf(`{
+		"ignition": {
+			"version": "$version",
+			"security": {
+				"tls": {
+					"certificateAuthorities": [{
+						"source": "http://127.0.0.1:8080/ca"
```

Comment:
Is this endpoint being served by any handler?  We want a `source` that includes the first CA but not the second, right?

Reply:
I was just following the above test case (https://github.com/coreos/ignition/pull/968/files#diff-1ec18742f0f11a112d5e166de55f0e71L163), but I feel we can improve this test case by adding a handler which will serve an invalid cert bundle.